src/components/nve-tab-group/nve-tab-group.component.ts

```diff
@@ -0,0 +1,237 @@
+import { html, LitElement } from 'lit';
+import { customElement, property, query } from 'lit/decorators.js';
+import { INveComponent } from '@interfaces/NveComponent.interface';
+import styles from './nve-tab-group.styles';
+import { NveTab, NveTabPanel } from 'src/nve-designsystem';
+
+/**
+ * En komponent for å vise faner. Bruker underkomponenter for å vise knapper og det innholdet den spesifikke knappen skal vise
+ *
+ * NveTab er selve knappen for å velge fane
+ * NveTabPanel er innholdet for hver fane
+ * Disse kobles sammen med `panel` på NveTab som skal være det samme som `name` på NveTabPanel
+ *
+ * @slot - For innholdet på hver fane. Må være en eller flere `<nve-tab-panel>`-elementer
+ * @slot nav - For fane-knapper. Må være av typen `<nve-tab>`
```

Comment:
Hvis det kun er nve-tab som kan være i dette sporet, trenger man egentlig å angi spor?
Kunne du ikke sjekket hovedsporet for hvilke elementer som var av typen nve-tab-panel?
Eksempel:
```html
<nve-tab-group showbackground>
  <nve-tab panel="p1">Fane 1</nve-tab>
  <nve-tab panel="p2">Fane 2</nve-tab>
  <nve-tab-panel name="p1">Dette er innholdet i den første fanen</nve-tab-panel>
  <nve-tab-panel name="p2">Innhold i fane nummer 2</nve-tab-panel>
</nve-tab-group>
```